internal/custom-code-logging/command_custom_code_logging.go

```diff
@@ -0,0 +1,73 @@
+package customcodelogging
+
+import (
+	"github.com/spf13/cobra"
+
+	cclv1 "github.com/confluentinc/ccloud-sdk-go-v2/ccl/v1"
+
+	pcmd "github.com/confluentinc/cli/v4/pkg/cmd"
+	"github.com/confluentinc/cli/v4/pkg/config"
+	"github.com/confluentinc/cli/v4/pkg/featureflags"
+	"github.com/confluentinc/cli/v4/pkg/output"
+)
+
+type customCodeLoggingCommand struct {
+	*pcmd.AuthenticatedCLICommand
+}
+
+type customCodeLoggingShortOut struct {
+	Id          string `human:"ID" serialized:"id"`
+	Cloud       string `human:"Cloud" serialized:"cloud"`
+	Region      string `human:"Region" serialized:"region"`
+	Environment string `human:"Environment" serialized:"environment"`
+}
+
+type customCodeLoggingOut struct {
+	Id          string `human:"ID" serialized:"id"`
+	Cloud       string `human:"Cloud" serialized:"cloud"`
+	Region      string `human:"Region" serialized:"region"`
+	Environment string `human:"Environment" serialized:"environment"`
+	Destination string `human:"Destination" serialized:"destination"`
+	Topic       string `human:"Topic" serialized:"topic"`
+	ClusterId   string `human:"Cluster ID" serialized:"cluster"`
+	LogLevel    string `human:"Log Level" serialized:"log_level"`
+}
+
+func New(cfg *config.Config, prerunner pcmd.PreRunner) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:         "custom-code-logging",
+		Short:       "Manage custom code logging.",
+		Annotations: map[string]string{pcmd.RunRequirement: pcmd.RequireNonAPIKeyCloudLogin},
+	}
+
+	c := &customCodeLoggingCommand{pcmd.NewAuthenticatedCLICommand(cmd, prerunner)}
+
+	_ = cfg.ParseFlagsIntoConfig(cmd)
+	if cfg.IsTest || featureflags.Manager.BoolVariation("cli.custom_code_logging.early_access", cfg.Context(), config.CliLaunchDarklyClient, true, false) {
```

Comment:
please work with cli team to create this flag, we can use this team to as the owner of this flag https://app.launchdarkly.com/settings/teams/flink-artifact-ea/members